kafka-ssl/register-postgres.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "inventory-connector",
+    "config": {
+        "connector.class": "io.debezium.connector.postgresql.PostgresConnector",
```

Comment:
Please use MySQL connector. PostgreSQL connector does not need internal schema history so the lines below would be irelevant.